index.html

```diff
@@ -17,7 +17,8 @@ <h2>Score</h2>
       <p>Pairs guessed: <span id="pairs-guessed">0</span></p>
     </div>
     <div id="memory-board"></div>
-
+    <script src="js/memory.js" type="text/javascript"></script>
+    <script src="js/index.js" type="text/javascript"></script>
     <!-- LINK THE JAVASCRIPT FILES HERE (keep in mind that the order in which you link them MATTERS) -->
   </body>
 </html>
```

Comment:
Great stuff Elia! 
The code organization looks good.
Please check the comments for potential improvements.